pymc3/distributions/continuous.py

```diff
@@ -4226,6 +4226,31 @@ class Interpolated(BoundedContinuous):
     ========  ===========================================
     Support   :math:`x \in [x\_points[0], x\_points[-1]]`
     ========  ===========================================
+    
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pymc3 as pm
```

Comment:
We should also import arviz to be safe. ArviZ has its own styles, but they are only made available to matplotlib when arviz is imported, the code would work if arviz has already been imported at some point during the doc generation (which is quite probable) but we should not rely on that.